nuxeo-features/nuxeo-admin-center/nuxeo-connect-update/pom.xml

```diff
@@ -74,6 +74,11 @@
       <artifactId>nuxeo-connect-client-wrapper</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
```

Comment:
Shouldn't this be added on `<scope>provided</scope>` instead of `test` ?

Reply:
`test` is the minimal scope here